Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -467,13 +467,9 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
         process.environment = ProcessInfo.processInfo.environment
 #if os(Windows)
         let pluginLibraryPath = self.toolchain.swiftPMLibrariesLocation.pluginLibraryPath.pathString
-        var env = ProcessInfo.processInfo.environment
-        if let Path = env["Path"] {
-            env["Path"] = "\(pluginLibraryPath);\(Path)"
-        } else {
-            env["Path"] = pluginLibraryPath
-        }
-        process.environment = env
+        var env = Environment(ProcessInfo.processInfo.environment)
```

Comment:
I think there's a Env.current you can use instead. No no need to use ProcessInfo